bin/parser/index.js

```diff
@@ -15,9 +15,28 @@ function parseChangelogItem(item) {
   };
 }
 
+function isUsableItem(potentialItem) {
+  return potentialItem.split('\n').length > 1;
+}
+
+function isChangelogCorrectlyFormatted(items) {
```

Comment:
Any issues with replacing this with `return items.every(isUsableItem)`? Not sure whether it is lazy or not, but the readability would be way worth it anyway imo.